pipelines.go

```diff
@@ -82,6 +82,14 @@ type PipelineList []struct {
 	WebURL string `json:"web_url"`
 }
 
+type SimplePipeline struct {
```

Comment:
The simple pipeline could be used in the ListPipeline struct too, as these are the same. That would break the API, and I don't know what the policy is around that pre 1.0 for this project.

Reply:
I don't like the name `SimplePipeline` as to me it indicates a type of pipeline. Could we use `PipelineInfo` or `PipelineMeta` of something alike? Updating`ListPipeline` to use this struct makes sense and should be fine.

Reply:
Picked PipelineInfo, as I felt it was more descriptive.